docs/topics/clusterdefinitions.md

```diff
@@ -578,6 +578,7 @@ A cluster can have 0 to 12 agent pool profiles. Agent Pool Profiles are used for
 | auditDEnabled                                                  | no                                                                   | Enable auditd enforcement at the OS layer for each node VM. This configuration is only valid on an agent pool with an Ubuntu-backed distro, i.e., the default "aks-ubuntu-16.04" distro, or the "aks-ubuntu-18.04", "ubuntu", "ubuntu-18.04", or "acc-16.04" distro values. Defaults to `false`                                                                                                                                                                                                                                                                                                                                                                                                                                           |
 | customVMTags                                                   | no                                                                   | Specifies a list of custom tags to be added to the agent VMs or Scale Sets. Each tag is a key/value pair (ie: `"myTagKey": "myTagValue"`).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| diskEncryptionSetID                                            | no                                                             | Specifies ResourceId of the disk encryption set to use for enabling encryption at rest (ie: `"/subscriptions/{subs-id}/resourceGroups/{rg-name}/providers/Microsoft.Compute/diskEncryptionSets/{diskEncryptionSet-name}"`).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
if this feature is not available in all regions can we make sure to point that out here for aks-engine users

Reply:
added an official link